app/Main.hs

```diff
@@ -208,6 +226,6 @@ main = do
       buildErrorMessages errors = do
         forM errors \(from, err) -> do
           let entryTo = getEntryFromCreateError err
-          let header = "'" +| from |+ "' to '" +| entryTo ^. path |+ "':"
+          let header = [int|s|'#{from}' to '#{entryTo ^. path}':|]
           let errorMsg = createErrorToBuilder err
           pure $ unlinesF @_ @Builder $ header : [indentF 2 errorMsg]
```

Comment:
I didn't test it, but would this work?

```hs
          let entryTo = getEntryFromCreateError err
          pure [int||
            '#{from}' to '#{entryTo ^. path}':
              #{createErrorToBuilder err}
          |]
```

Reply:
Actually, also just noticed that `buildErrorMessages` doesn't have to return `Sem r [Builder]`, it could just return a `[Builder]`.

Reply:
Oh, I'm afraid no. Just a couple of days ago I witnessed the need for such functionality too, so let the ticket be https://github.com/serokell/nyan-interpolation/issues/11.

Reply:
Ah right, and one of the messages returned by `createErrorToBuilder` does have multiple lines :/

Ok, let's leave it like this for now. @DK318, please create a separate issue to improve this when https://github.com/serokell/nyan-interpolation/issues/11 is done + released.

Reply:
Note: while we're here, we could still change `Sem r [Builder]` -> `[Builder]` in this PR.

Reply:
@DK318 I checked, `Sem r [Builder]` can be changed to `[Builder]` (as mentioned above). For example, like this:
```
buildErrorMessages = fmap \(from, err) ->
  let entryPath = getEntryFromCreateError err
      header    = [int|s|'#{from}' to '#{entryTo ^. path}':|]
      errorMsg  = createErrorToBuilder err
  in unlinesF @_ @Builder $ header : [indentF 2 errorMsg]
```